oak_attestation_verification/src/verifier.rs

```diff
@@ -73,8 +87,30 @@ pub fn verify_internal(
     }
 }
 
-fn verify_certificate_chain(_evidence: &Evidence) -> anyhow::Result<()> {
-    anyhow::bail!("Needs implementation")
+fn verify_certificate_chain(evidence: &Evidence) -> anyhow::Result<()> {
```

Comment:
I think we could rename it to `verify_dice_certificate_chain` to be explicit

Reply:
Going with verify_dice_chain.